### PR TITLE
Add version 3.3.0 to MSTK spackage

### PR DESCRIPTION
**Pull request by raovgarimella**

### CODE REVIEW

File: var/spack/repos/builtin/packages/mstk/package.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/mstk/package.py]: Can you revert the change to the copyright?

Comment (raovgarimella) [reply var/spack/repos/builtin/packages/mstk/package.py]: Done
